TeamCode/src/main/java/org/firstinspires/ftc/teamcode/Robot.java

```diff
@@ -18,9 +20,11 @@ public static Robot getInstance() {
     }
 
     //add subsystems...
+    public Claw CLAW;
```

Comment:
vlaw is not a constant, consider making it a 'claw'

Reply:
זרום יטוסיק זה מגניב יותר

Reply:
שמע לא כזה משנה לי זה FTC וזה מצחיק רצח אז תעשו מה שבא לכם